app/controllers/users_controller.rb

```diff
@@ -2,12 +2,16 @@
 class UsersController < ApplicationController
   load_permissions_and_authorize_resource
   before_action :set_user
+
   def index
   end
 
   def show
   end
 
+  def profile
```

Comment:
Haha.
I just wanted to differ from `/profile` (where you edit your own user-profile-information)
to `/anvandare/45` where you look at a specific user.